### ISSUE TITLE
Add fallbacks to content negotiation

### ISSUE THREAD

**Issue by kipparker** [icebreakerone/registry#2]
The lambda returns the correct file eg. xml, jsonld, but it doesn't have a default so eg. /registry without an Accepts header just returns a 404. We'll need to change it so that html is the default, which means we'll also need to add a "does the file exist" check

**Comment by croesus** [comment]
Does it return 415 (unsupported media type) if we don't have the representation the Accepts header is asking for?

**Comment by kipparker** [comment]
> Does it return 415 (unsupported media type) if we don't have the representation the Accepts header is asking for?

No, it would just return html as that's the fallback - we can add 415 if you like?
